### ISSUE TITLE
[POST-REBASE] no resource type if single type resource have multiple items

### ISSUE THREAD

**Issue by guangxuli** [openshift/origin#13450]
**provide a description of the issue**

not show resource type when run the command **oc get all**  if there are multiple items of single type resource.

##### Version
[provide output of the `openshift version` or `oc version` command]
[cloud@centos kube]$ oc version
oc v1.5.0-alpha.2+4aecfcc-601
kubernetes v1.5.2+43a9be4
features: Basic-Auth

Server https://192.168.122.251:8443
openshift v1.5.0-alpha.2+4aecfcc-601
kubernetes v1.5.2+43a9be4

##### Steps To Reproduce

```
[cloud@centos kube]$ oc get all
NAME                  TYPE      FROM        LATEST
bc/ruby-hello-world   Docker    Git@beta4   1

NAME                        TYPE      FROM        STATUS                         STARTED   DURATION
builds/ruby-hello-world-1   Docker    Git@beta4   New (InvalidOutputReference)             

NAME                  DOCKER REPO   TAGS      UPDATED
is/ruby-20-centos7                  latest    58 seconds ago
is/ruby-hello-world                           

NAME                  REVISION   DESIRED   CURRENT   TRIGGERED BY
dc/ruby-hello-world   0          1         0         config,image(ruby-hello-world:latest)

NAME                   CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
svc/ruby-hello-world   172.30.30.148   <none>        8080/TCP   1m

```

```
[cloud@centos kube]$ oc delete bc/ruby-hello-world dc/ruby-hello-world svc/ruby-hello-world
buildconfig "ruby-hello-world" deleted
deploymentconfig "ruby-hello-world" deleted
service "ruby-hello-world" deleted
```

```
[cloud@centos kube]$ oc get all
NAME               DOCKER REPO   TAGS      UPDATED
ruby-20-centos7                  latest    About a minute ago
ruby-hello-world
```

if we did like this that keep one imagestream stay:

```
[cloud@centos kube]$ oc get all
NAME                  TYPE      FROM        LATEST
bc/ruby-hello-world   Docker    Git@beta4   1

NAME                        TYPE      FROM        STATUS                         STARTED   DURATION
builds/ruby-hello-world-1   Docker    Git@beta4   New (InvalidOutputReference)             

NAME                  DOCKER REPO   TAGS      UPDATED
is/ruby-20-centos7                  latest    8 seconds ago
is/ruby-hello-world                           

NAME                  REVISION   DESIRED   CURRENT   TRIGGERED BY
dc/ruby-hello-world   0          1         0         config,image(ruby-hello-world:latest)

NAME                   CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
svc/ruby-hello-world   172.30.120.152   <none>        8080/TCP   9s
```
```
[cloud@centos kube]$ oc delete svc/ruby-hello-world dc/ruby-hello-world is/ruby-hello-world  bc/ruby-hello-world
service "ruby-hello-world" deleted
deploymentconfig "ruby-hello-world" deleted
imagestream "ruby-hello-world" deleted
buildconfig "ruby-hello-world" deleted
```

```
[cloud@centos kube]$ oc get all
NAME                 DOCKER REPO   TAGS      UPDATED
is/ruby-20-centos7                 latest    34 seconds ago
```

##### Current Result
no resource type if single type resource have multiple items
##### Expected Result
should show resource type if there are multiple items

**Comment by mfojtik** [comment]
@fabianofranz not sure if this is a regression from introducing api groups or not :)

**Comment by fabianofranz** [comment]
@mfojtik no, doesn't seem so.
@juanvallejo fyi

**Comment by guangxuli** [comment]
@mfojtik @fabianofranz Seems that @juanvallejo have fix this in https://github.com/kubernetes/kubernetes/pull/37636.

**Comment by fabianofranz** [comment]
Right, waiting for the next rebase.

**Comment by fabianofranz** [comment]
Fixed by the 1.6 Kubernetes rebase.
